gsrs-core-entities/src/main/java/ix/core/models/UserProfile.java

```diff
@@ -195,7 +195,7 @@ public boolean isRoleQueryOnly(){
 	}
 	
 	public UserProfile standardize() {
```

Comment:
I think similar issues may happen with calls here. May need to change this to be analogous too?

Reply:
I thought about that but the places where that is called are in Transactions where we might already be creating a new user so I was less concerned about it